src/assets/text/helloworld.nf

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nextflow
+
+// tutorial.nf example from https://www.nextflow.io/docs/latest/getstarted.html
+
```

Comment:
Ditto for authorship and description
https://docs.dockstore.org/en/develop/advanced-topics/best-practices/nfl-best-practices.html?highlight=metadata#authorship-metadata